packages/pubsub/src/subscription.js

```diff
@@ -1042,9 +1071,16 @@ Subscription.prototype.setMetadata = function(metadata, gaxOpts, callback) {
     gaxOpts = {};
   }
 
+  var body = Subscription.formatMetadata_(metadata);
```

Comment:
Can we rename this `subscription`? We usually use `body` to mean "JSON payload" of a request.

Reply:
Done!